src/sonic-yang-models/yang-models/sonic-buffer-pg.yang

```diff
@@ -0,0 +1,66 @@
+module sonic-buffer-pg {
+	namespace "http://github.com/Azure/sonic-buffer-pg";
+	prefix bpg;
+
+	import sonic-extension {
+		prefix sonic-ext;
+	}
+
+	import sonic-port {
+		prefix prt;
+	}
+
+	import sonic-buffer-profile {
+		prefix bpf;
+	}
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC BUFFER PG";
+
+	revision 2019-05-15 {
+		description
+			"Initial revision.";
+	}
+
+
+	container sonic-buffer-pg {
+
+		container BUFFER_PG {
+
+			list BUFFER_PG_LIST {
+				key "ifname pg_num";
+				//sonic-ext:key-pattern "BUFFER_PG|{ifname}|{pg_num}"; //special pattern used for extracting keys from
+				//redis-key and fill populate the yang instance
+				// Total list instance = number(key1) * number(key2) * number(key3)
+
+				leaf ifname {
```

Comment:
if logical interfaces are not allowed and this is always physical port, then better to use port_name\port. Agree?

Reply:
changed to port.